### ISSUE TITLE
Sticky div doesn't stick on iPad instead it disappears and it is visible after few seconds

### ISSUE THREAD

**Issue by dolphine4u** [leafo/sticky-kit#166]

**Comment by pacMakaveli** [comment]
Same issue here. It works fine on desktop emulated viewport ( Chrome ) but on actual device it disappears on scroll and reappears once stopped. However, it does scroll, it's just that initial scrolling phase where it's not visible.

**Comment by liormb** [comment]
Same here. @dolphine4u @pacMakaveli do you have a fix for it?

**Comment by pacMakaveli** [comment]
@liormb yes, adding `.sticky-in-parent { transform: translate3d(0, 0, 0); }` where `.sticky-in-parent` is my sticky-kit selector. It doesn't completely solves the problem as I've noticed from time to time where it does it again, but only if you scroll REALLY FAST! It's better than nothing though.

**Comment by liormb** [comment]
@pacMakaveli, sounds good, I'll give it a try. Thanks.

**Comment by Esenor** [comment]
@pacMakaveli great solve my problems

**Comment by robbyparker** [comment]
The CSS trick didn't work for me. Any other suggestions by anyone?

**Comment by Webtransformer** [comment]
Same problem here. The CSS trick didn´t worked for me.

**Comment by jimmybrawn** [comment]
We need a cure please...

**Comment by tameshwari** [comment]
Please provide a solution for this. Do some needful.

**Comment by glennabaron** [comment]
I had an issue with this and resolved it by setting the `spacer` option to false, in addition to setting `transform: translate3d(0, 0, 0);` on the sticky element.

Hopefully that helps someone.
